<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="InSalir"/>
    <addaction name="inGuardar"/>
    <addaction name="separator"/>
    <addaction name="inSalir"/>
   </widget>
   <widget class="QMenu" name="menuForma">
    <property name="title">
     <string>Forma</string>
    </property>
    <addaction name="InLineas"/>
    <addaction name="InLibre"/>
    <addaction name="InRectang"/>
    <addaction name="InCircuferencia"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuForma"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="InSalir">
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="inGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="inSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionAncho">
   <property name="text">
    <string>Ancho</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="InLineas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lineas</string>
   </property>
  </action>
  <action name="InLibre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Libre</string>
   </property>
  </action>
  <action name="InRectang">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rect√°ngulos</string>
   </property>
  </action>
  <action name="InCircuferencia">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Circunferencias</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
